lib/stripe.rb

```diff
@@ -46,49 +48,36 @@
 require "stripe/oauth"
 
 module Stripe
-  DEFAULT_CA_BUNDLE_PATH = __dir__ + "/data/ca-certificates.crt"
-
   @app_info = nil
 
-  @api_base = "https://api.stripe.com"
-  @connect_base = "https://connect.stripe.com"
-  @uploads_base = "https://files.stripe.com"
-
-  @log_level = nil
-  @logger = nil
-
-  @proxy = nil
-
-  @max_network_retries = 0
-  @max_network_retry_delay = 2
-  @initial_network_retry_delay = 0.5
-
-  @ca_bundle_path = DEFAULT_CA_BUNDLE_PATH
-  @ca_store = nil
-  @verify_ssl_certs = true
-
-  @open_timeout = 30
-  @read_timeout = 80
-
-  @enable_telemetry = true
+  @configuration = Stripe::StripeConfiguration.setup
```

Comment:
We could probably expose this if there's ever a desire to permit something like:
```
Stripe.configure do |config|
  # config.read_timeout = 100
end
```

Reply:
Good point! Let's expose it separately if we decided to do so, but I think this configuration style would be much more "Ruby-ish" than what the gem currently offers.